### PR TITLE
(WIP) Rename ARMs to PRFCs

### PR DESCRIPTION
**Pull request by ahpook**
This is a work-in-progress pull request that arose from comments in #70 - renaming the ARM process to Puppet RFC, abbreviated PRFC for disambiguation with the IETF's "real" RFCs.

I'm putting the pull request up in this initial form to see if the wording, "flow" and aesthetics of the new name make sense and folks consider them an improvement.  If so, I'll keep going; I'll also happily take PRs against my branch to convert the deeper ARMs. (I've renamed the directories and index references here but haven't gone through search-and-replace on the text or metadata.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+---------------------
 
-<blockquote>
-armature |ˈärməCHər, -ˌCHo͝or |<br />
-2 a metal framework on which a sculpture is molded with clay or similar material. <br />
-• a framework or formal structure, esp. of a literary work: <i>Shakespeare's plots have served as the armature for many novels.</i><br />
--- <b>New Oxford American Dictionary</b>
-</blockquote>
+This repository contains Puppet RFCs, which are "Requests For Comments" in the tradition of the IETF 
+RFC process. They are proposals to enhance / add features to Puppet or something in the Puppet
```
Comment (igalic) [inline README.md]: actually, this *does* flow very nicely.

File: prfc-0.prfc/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+   "prfc":0,
+   "title":"Puppet RFC Process",
+   "champion":"hlindberg",
```
Comment (igalic) [inline prfc-0.prfc/metadata.json]: this still uses champion, something that in a previous commit was renamed.

**Comment by hlindberg** [comment]
I find PRFC slightly (just one notch above the level were I would not have bothered to comment) hard to pronounce (and hence to read), first attempting something like "prrffs" until realizing it has the letters RFC in it. Writing it pRFC or P-RFC would make it easier. OTOH, it is probably just a matter of getting familiar with the acronym and knowing how it is pronounced.

**Comment by ferventcoder** [comment]
@hlindberg I'm practicing `PRFC` as "Puppet RFC"

File: prfc-0.prfc/metadata.json (hunk 2 above)
Comment (ahpook) [inline prfc-0.prfc/metadata.json]: good catch @igalic , fixed.

**Comment by ahpook** [comment]
PR! (*clap clap*) FC! (*clap clap*) (for the Portland Timbers Football Club fans)

File: prfc-0.prfc/prfc.md [hunk 3]
```diff
@@ -250,8 +245,8 @@ particular reviewer's name.
 Finally
 -------
 
-Happy ARM-wrestling, and remember [we welcome all
-feedback](https://github.com/puppetlabs/armatures/issues/new) on the process.
+Happy commenting, and remember [we welcome all
+feedback](https://github.com/puppetlabs/puppet-rfc/issues/new) on the process.
```
Comment (ferventcoder) [inline prfc-0.prfc/prfc.md]: Renaming the repo after this is accepted?

Comment (ahpook) [inline prfc-0.prfc/prfc.md]: yeah that was the idea.

Comment (ferventcoder) [inline prfc-0.prfc/prfc.md]: :+1:

**Comment by jdwelch** [comment]
I'm :+1: on this; Anything left before merge?

**Comment by ahpook** [comment]
Looks pretty unanimous, let's do it and rename the repo.

**Pull request merged**